ext/windows/run_facter_interactive.bat

```diff
@@ -0,0 +1,7 @@
+@echo Running facter on demand ...
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+elevate.exe "%~dp0facter_interactive.bat"

```

Comment:
`facter.bat` and `facter_interactive.bat` work fine, but I'm having trouble with `run_facter_interactive.bat`. It seems like the call to `elevate.exe` isn't right. Any thoughts?

cc/ @joshcooper @MikaelSmith